src/REST/Section/Project.php

```diff
@@ -63,4 +63,31 @@ public function listVersions($project) : array
     {
         return $this->Jira->get("project/{$project}/versions");
     }
+
+    /**
+     * Returns latest version in project<br>
```

Comment:
<br> :)

Reply:
Just try to open Quick dock in PHPStorm without `<br>`=))